src/index.js

```diff
@@ -309,11 +312,12 @@ class ReactNativeModal extends Component {
 
   handleDimensionsUpdate = dimensionsUpdate => {
     // Here we update the device dimensions in the state if the layout changed (triggering a render)
-    const deviceWidth = Dimensions.get("window").width;
-    const deviceHeight = Dimensions.get("window").height;
+    const { deviceHeight, deviceWidth } = this.props;
```

Comment:
And lastly, here, if the user has supplied it's own `deviceWidth` and `deviceHeight` props we just ignore updating the dimensions (this should be handled directly by the user, outside of the component).